src/imitation/scripts/parallel.py

```diff
@@ -70,7 +85,7 @@ def parallel(
     try:
         ray.tune.run(
             trainable,
-            config=search_space,
+            config=copy.deepcopy(search_space),
```

Comment:
Again, a comment explaining this would be useful.

Reply:
(see explanation above in https://github.com/HumanCompatibleAI/imitation/pull/194#discussion_r434180592)